src/transformers/models/conv_bert/modeling_conv_bert.py

```diff
@@ -0,0 +1,1261 @@
+# coding=utf-8
+# Copyright 2021 The HuggingFace Team The HuggingFace Inc. team. All rights reserved.
```

Comment:
I think this will apply to all copyrights

Reply:
done